spec/spec_helper.rb

```diff
@@ -13,6 +13,6 @@
   config.order = 'random'
 
   def env(opts={})
-    @_env ||= {}
+    @_env ||= 'I am env'
```

Comment:
I would prefer that spec helper env would look like real env since it is very general and somebody is bound to change it to be general here sooner or later. So instead of 'I am env', use something like: ```{ mock_service: :mock_value }``` or similar